Makefile

```diff
@@ -0,0 +1,25 @@
+##############################################################
+#               CMake Project Wrapper Makefile               #
+############################################################## 
+
+SHELL := /bin/bash
+RM    := rm -rf
```

Comment:
That's the common rm-command in every standard Makefile. But I can delete the "force" option if you like.